Description: Suppress "File is write protected" message
 This isn't useful information, and is rather confusing as it may be
 interpreted as an error rather than just a random fact about a flag set in
 the file, so silence this message.
Author: Olly Betts <olly@survex.com>
Forwarded: no
Last-Update: 2014-10-21

--- a/src/xlsparse.c
+++ b/src/xlsparse.c
@@ -145,7 +145,7 @@
 		break;
 	}
 	case WRITEPROT: {
-		fprintf(stderr,"File is write protected\n");
+		/*fprintf(stderr,"File is write protected\n"); */
 		break;
 	}
 		
